extensions/machine-learning-services/package.nls.json

```diff
@@ -5,8 +5,8 @@
 	"title.documents": "Documents",
 	"title.configurations": "Configurations",
 	"title.endpoints": "Endpoints",
-	"mls.command.managePackages": "Manage Packages in SQL Server",
-	"mls.command.manageLanguages": "Manage External Languages",
+	"mls.command.managePackages": "Manage packages in database",
+	"mls.command.manageLanguages": "Manage external languages",
 	"mls.command.predictModel": "Make prediction",
 	"mls.command.manageModels": "Manage models",
```

Comment:
little outside the scope... manage models or deploy models?

Reply:
still trying t figure that out. manage model  includes deploy and predict 


---
In reply to: [393904856](https://github.com/microsoft/azuredatastudio/pull/9519#discussion_r393904856) [](ancestors = 393904856)